### ISSUE TITLE
Distinguish build error from faild deploy

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#530]
@ctrueden Do you think it would be possible (and maybe in general a good idea) to separate the `build` and the `deploy to scijava maven` into **two steps** within the github actions?

The issue is that right now one needs to scroll all the way down the `Execute the build` step to find out that the actual `build` and all the tests worked, but that it just failed to deploy to maven because the version already existed: https://github.com/mobie/mobie-viewer-fiji/runs/4189660027?check_suite_focus=true

**Comment by constantinpape** [comment]
See also https://github.com/mobie/mobie-viewer-fiji/issues/497#issuecomment-968045546

**Comment by ctrueden** [comment]
@tischi Yes, good idea. If someone tweaks this in `github-actionify.sh` and files a PR, I would then merge and use it to convert all the repositories for which I have push access. (The `github-actionify.sh` script is intended to
